proposals/lip-0052.md

```diff
@@ -133,10 +133,10 @@ The following constants are used throughout the document:
 | `NFT_NOT_LOCKED` | string | `MODULE_NAME_NFT` |
 | `ALL_SUPPORTED_NFTS_KEY` | bytes | `EMPTY_BYTES` |
 | **NFT Store Constants** | | |
-| `SUBSTORE_PREFIX_NFT` | bytes | 0x00 00 |
-| `SUBSTORE_PREFIX_USER` | bytes | 0x80 00 |
+| `SUBSTORE_PREFIX_NFT` | bytes | 0x80 00 |
+| `SUBSTORE_PREFIX_USER` | bytes | 0x40 00 |
 | `SUBSTORE_PREFIX_ESCROW` | bytes | 0xc0 00 |
-| `SUBSTORE_PREFIX_SUPPORTED_NFTS` | bytes | 0xd0 00 |
+| `SUBSTORE_PREFIX_SUPPORTED_NFTS` | bytes | 0x20 00 |
```

Comment:
@ricott1 currently in SDK we pass index 4 to supportedNFTs subStore for which computed value is `2000` using [this function](https://github.com/LiskHQ/lisk-sdk/blob/98c3d9c588b56845e46eb8a209985a92bf478c67/framework/src/modules/base_store.ts#L35)
https://github.com/LiskHQ/lisk-sdk/blob/8a2dec0d48b9ce1f688717a6649e9686ce64475a/framework/src/modules/nft/module.ts#L119-L122

Reply:
@ishantiw In NFT, the index should start from `0` instead of `1`.  It should be :
NFTStore: `0`
UserStore: `1`
EscrowStore: `2`
SupportedNFTStore: `3`